packages/react-native/scripts/cocoapods/rndependencies.rb

```diff
@@ -56,24 +56,37 @@ def add_rn_dependencies(s)
 end
 
 class ReactNativeDependenciesUtils
-    @@build_from_source = false
+    @@build_from_source = true
     @@react_native_path = ""
     @@react_native_version = ""
 
     def self.build_react_native_deps_from_source()
         return @@build_from_source
     end
 
+    def self.podspec_source()
```

Comment:
Maybe we could call this `resolve_podspec_source` to indicate that it **does something** - not only **returns the value**?

Reply:
it really doesn't do anything. It just returns the source value based on some conditions.

Reply:
It might end up calling `self.podspec_source_download_prebuild_release_tarball()` which ends up downloading both `release` and `debug` prebuilds.